go.sum

```diff
@@ -261,6 +261,8 @@ github.com/sourcegraph/jsonx v0.0.0-20200629203448-1a936bd500cf h1:oAdWFqhStsWii
 github.com/sourcegraph/jsonx v0.0.0-20200629203448-1a936bd500cf/go.mod h1:ppFaPm6kpcHnZGqQTFhUIAQRIEhdQDWP1PCv4/ON354=
 github.com/sourcegraph/sourcegraph/lib v0.0.0-20210914094904-648e35b48ded h1:otRtA0pYQ1CzcqQ/XJjPJ3HNAVP+MqyMQN5i0eUX8mA=
 github.com/sourcegraph/sourcegraph/lib v0.0.0-20210914094904-648e35b48ded/go.mod h1:9vLg5fQL9Q3iktfqu96OrswbOq7yqMesMJER9DCy8vk=
```

Comment:
Is this still required?

Reply:
`go mod tidy` says no, so it's gone now.